stylesheets/_component.datatables.scss

```diff
@@ -101,6 +102,12 @@ tr.selected .table-row-select::before {
 // scss-lint:disable SelectorFormat
 th.sorting_disabled:hover {
     text-decoration: none;
+
+    > a {
+        border: 0;
+        cursor: default;
```

Comment:
Any reason why this has been added back in? Doesn't play nice with PHP fallbacks.